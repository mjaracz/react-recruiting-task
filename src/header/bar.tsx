import React, { FC } from 'react'
import {useStyle} from './styled'
import {Link} from 'react-router-dom'

export const HeaderBar: FC = () => {
  const {root, barItem} = useStyle()
  return (
    <nav className={root}>
      <Link to="/absences" className={barItem}>absences</Link>
      <Link to="/members" className={barItem}>members</Link>
    </nav>
  )
}