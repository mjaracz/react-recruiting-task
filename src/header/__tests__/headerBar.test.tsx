import React from 'react'
import {render} from '@testing-library/react'
import {HeaderBar} from '../bar'
import {BrowserRouter} from 'react-router-dom'

describe('<HeaderBar/>', () => {
  it('should be defined and match snapshot', function () {
    const { baseElement } = render(<HeaderBar/>, { wrapper: BrowserRouter })
    expect(baseElement).toMatchSnapshot()
  });
})