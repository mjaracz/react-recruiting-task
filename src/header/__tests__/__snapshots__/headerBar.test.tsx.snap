// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderBar/> should be defined and match snapshot 1`] = `
<body>
  <div>
    <nav
      class="makeStyles-root-1"
    >
      <a
        class="makeStyles-barItem-2"
        href="/absences"
      >
        absences
      </a>
      <a
        class="makeStyles-barItem-2"
        href="/members"
      >
        members
      </a>
    </nav>
  </div>
</body>
`;
